Tests/Marketplace/marketplace_services.py

```diff
@@ -1659,7 +1659,7 @@ def collect_content_items(self):
             logging.exception(f"Failed collecting content items in {self._pack_name} pack")
         finally:
             self._content_items = content_items_result
-            return task_status, content_items_result
+            return task_status
```

Comment:
don't we need the other return value?
wouldn't it be better to fix the condition in the calling function?
```python
if not results[0]:
    print('error')

Reply:
Nope, we set the value in the line before as an attribute of the Pack object and use it later on, therefore it is not needed.